.travis.yml

```diff
@@ -1,7 +1,13 @@
+dist: trusty
```

Comment:
I feel like I'm missing something in the output from travis that drove this change. But can you edit the commit message to indicate why you needed to move to trusty?  It might save us time if we feel the need to move forward and see issues.  Then again it might not, but I'm not sure what drove the change. ;)